docs/source/faq.rst

```diff
@@ -319,6 +319,37 @@ frequently seem to work with the Enthought Canopy Python
 distribution. However, we recommend compiling SimpleITK
 explicitly against this Python distribution to ensure compatibility.
 
+How do I read an image with a special character in the filename?
+~~~~~~~~~~~~~~~~
+
+Special characters are non-ANSI characters, which require a non-ASCII encoding to for representation. Unicode is the
+modern standard with UTF-8 being the most common. The underlying ITK API call use bytes to represent the filename which
```

Comment:
call->calls: "The underlying ITK API call use bytes" -> "The underlying ITK API calls use bytes" 
or
use->uses: "The underlying ITK API call use bytes" -> "The underlying ITK API call uses bytes"